akka-projection-grpc/src/main/mima-filters/1.4.0-M1.backwards.excludes/replicated-es-over-grpc.excludes

```diff
@@ -0,0 +1,13 @@
+# internal protocol class changes
```

Comment:
Funky with the required directory names here, MiMa would not pick it up if I put it in 1.3.1.backwards.excludes for some reason